apps/etterlatte-beregning/src/main/kotlin/avkorting/AvkortingService.kt

```diff
@@ -19,88 +22,124 @@ class AvkortingService(
         behandlingId: UUID,
         brukerTokenInfo: BrukerTokenInfo,
     ): Avkorting? {
+        tilstandssjekk(behandlingId, brukerTokenInfo)
         logger.info("Henter avkorting for behandlingId=$behandlingId")
         val behandling = behandlingKlient.hentBehandling(behandlingId, brukerTokenInfo)
+        val eksisterendeAvkorting = avkortingRepository.hentAvkorting(behandling.id)
 
-        return if (behandling.behandlingType == BehandlingType.FØRSTEGANGSBEHANDLING) {
-            hentAvkorting(behandling)
+        if (behandling.behandlingType == BehandlingType.FØRSTEGANGSBEHANDLING) {
+            return eksisterendeAvkorting?.let { avkortingMedTillegg(it, behandling) }
```

Comment:
Henger ikke helt med på denne. Dersom det er førstegangsbehandling så trenger vi ikke å gjøre avkorting på nytt dersom beregning er endret? Hva om saksbehandler går hele veien, så tilbake å endrer trygdetid for så å beregne på nytt i en førstegangsbehandling?

Reply:
Ja du har rett jeg. Jeg har bare glemt å gjøre det her.